generators/generator.h

```diff
@@ -65,6 +69,8 @@ namespace rpwa {
 
 		rpwa::Beam _beam;
 		rpwa::Target _target;
+		double _massX;
+		double _tPrime;
```

Comment:
properly move after `_vertex` to keep the same ordering as for the methods, also rearrange the initializer list accordingly.